napari/_qt/dialogs/preferences_dialog.py

```diff
@@ -154,7 +157,7 @@ def restore_defaults(self):
         widget.valueChanged.connect(self._reset_widgets)
         widget.exec_()
 
-    def _reset_widgets(self):
+    def _reset_widgets(self, disabled_plugins):
```

Comment:
@tlambert03  I think for me it was easier for me to know which plugins to re-enable when I clicked on `restore` defaults by taking the set stored in SETTINGS (that deep copy I was emitting) and unblocking those.  I agree it would be cleaner to go to the plugin manager to find out what is disabled and re-enable.  I can make that change so this is not as confusing.  Is there a quick way to re-enable everything?  I will look at it more to figure it out, but if you can give direction, I'd appreciate it!  

I think the source of truth *should* be the plugin manager.  That actually enables/disables plugins. At this point, SETTINGS really just needs to know the current state of things to be able to persist across sessions.  Is that what you're thinking?

Reply:
you know what?  I think I can just go through and set each plugin and set blocked as false.  there aren't that many.  that's my plan unless there is a better way.

Reply:
I've changed it to not rely on the settings value to reset the plugins.